ansible/roles/test/files/ptftests/arista.py

```diff
@@ -423,6 +426,25 @@ def verify_neigh_lag_state(self, lag, state="connected", pre_check=True):
         self.fails.add('%s: Invalid interface name' % msg_prefix[pre_check])
         return self.fails, lag_state
 
+    def verify_neigh_lag_no_flap(self):
+        is_flap = True
+        flap_cnt = -1
+        output = self.do_cmd('show interfaces Po1 | json')
+        if 'Invalid' not in output:
+            data = '\n'.join(output.split('\r\n')[1:-1])
+            obj = json.loads(data)
+
+            if 'interfaces' in obj and 'Port-Channel1' in obj['interfaces']:
+                intf_cnt_info = obj['interfaces']['Port-Channel1']['interfaceCounters']
+                flap_cnt = intf_cnt_info['linkStatusChanges']
+                is_flap = (flap_cnt != 0)
```

Comment:
I don't feel that having is_flap is absolutely necessary. Caller can deduce this information from flap_count since the assumption is that the counter is reset at the beginning of the test.

Reply:
done